cognitedata/reveal
# feat(react-components): add hotkeys P to toggle the Poi tool and Esc to remove the Poi creation - including the floating panel

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/

Add hotkeys for Point of Interests. 
P to toggle the tool and Esc to close/exit the creation

## Test instructions :information_source:

1. In Search UI, with a Scene loaded, press P to toggle the Poi tool from the left toolbar. 
2. Observe the tool is in enabling/disabling state.
3. Click on a 3d geometry to start the Point of Interest creation.
4. Focus the floating panel that has the Poi's description by typing in the description field or clicking the panel itself.
5. Press Escape button from the keyboard. 
6. Observe the creation, including the floating panel, is removed from the view and the tool is disabled.

## Checklist :ballot_box_with_check:
